datahub/core/log_config.py

```diff
@@ -0,0 +1,26 @@
+"""Dict configuration for formal logging."""
+LOG_LEVEL: str = "DEBUG"
```

Comment:
Ideally this should be configurable. Maybe make it so an environment variable can overwrite it?
```suggestion
import os

LOG_LEVEL: str = os.environ.get("API_LOG_LEVEL", "DEBUG")
```